modules/user/user.service.js

```diff
@@ -139,9 +142,7 @@ class UserService {
     const token = generateToken(user._id, user.email);
 
     return {
-      user: {
-        ...user._doc,
-      },
+      ...user._doc,
```

Comment:
Because we have User type, which is an output of the loginAdmin function and this type doesn't have such a field user. Instead of this, we can return all the necessary data about the admin

Reply:
l made a mistake